### ISSUE TITLE
Reduce unnecessary string creation when interact with rocksdb

### ISSUE THREAD

**Issue by wu-hanqing** [opencurve/curve#1583]

**Comment by zhanghuidinah** [comment]
@ccmaxcc Are you having some trouble? If so, please let us know so that we can communicate together.

**Comment by wuhongsong** [comment]
@ccmaxcc

**Comment by liam-thunder** [comment]
Interested in CurveFS, is this PR still available to contribute? I can start with this PR to get familiar with CurveFS.

**Comment by ilixiaocui** [comment]
> Interested in CurveFS, is this PR still available to contribute? I can start with this PR to get familiar with CurveFS.

@liam-thunder You are very welcome to contribute to this PR.

**Comment by liam-thunder** [comment]
address this issue in https://github.com/opencurve/curve/pull/2183

**Comment by Cyber-SiKu** [comment]
@liam-thunder Is this still going on?

**Comment by liam-thunder** [comment]
> @liam-thunder Is this still going on?

Sorry for late response, this issue is blocked by E2E benchmark testing and performance analysis, and requiring some extra efforts. But I'm quite busy recently, will get back on this issue later.

**Comment by Cyber-SiKu** [comment]
@liam-thunder Are you still going on?

**Comment by liam-thunder** [comment]
> @liam-thunder Are you still going on?

Yes, I'm working on deploy a single node cluster to do the benchmark.

**Comment by ilixiaocui** [comment]
> > @liam-thunder Is this still going on?
> 
> Sorry for late response, this issue is blocked by E2E benchmark testing and performance analysis, and requiring some extra efforts. But I'm quite busy recently, will get back on this issue later.


If you have any questions during the benchmarking process, please feel free to ask. You can also communicate in the WeChat group.
